reloj-checador/app/views/companies/index.html.erb

```diff
@@ -1,80 +1,78 @@
+                <button data-bs-toggle="modal" data-bs-target="#modal_a_e" onclick="edit('<%=company.id%>','<%=company.name%>',' <%=company.address%>')" title="Edit company" class="btn btn-primary"><i class="fas fa-edit"></i></button>
+                <%= link_to '', company_path(company), method: :delete, data: { confirm: 'Are you sure?'}, class: 'btn btn-danger fas fa-trash-alt', title:'Delete company' %>
+              </td>
+            </tr>
+          <%end%>
+        </tbody>
+      </table>
+      <div class="d-flex justify-content-center align-items-center mt-5">
+        <%= pagy_bootstrap_nav(@pagy).html_safe %>
+      </div>
+    </div>
+  </div>
 </div>
 
 <%= render 'modal_a_e', company: @company %>
 
 <script>
-    let name_modal = document.getElementById('name_modal');
-    let form_modal = document.getElementById("form_modal");
-    let name_edit = document.getElementById('company_name');
-    let address_edit = document.getElementById('company_address');
-    let type_method = document.getElementById('method_put');
-    let method = '<input type="hidden" name="_method" value="put" />';
-    let button = document.getElementById('buton_action_modal');
+  let name_modal = document.getElementById('name_modal');
+  let form_modal = document.getElementById("form_modal");
+  let name_edit = document.getElementById('company_name');
+  let address_edit = document.getElementById('company_address');
+  let type_method = document.getElementById('method_put');
+  let method = '<input type="hidden" name="_method" value="put" />';
+  let button = document.getElementById('buton_action_modal');
 
-    function add()
-    {
-        name_modal.innerHTML= 'Add Company';
-        form_modal.action = '/companies'
-        name_edit.value = "";
-        address_edit.value = "";
-        type_method.innerHTML ="";
-        button.classList.remove('d-none');
-    }
+  function add(){
+    name_modal.innerHTML= 'Add Company';
+    form_modal.action = '/companies'
+    name_edit.value = "";
+    address_edit.value = "";
+    type_method.innerHTML ="";
+    button.classList.remove('d-none');
+  }
 
-    function edit(id,name,address)
-    {
-        name_modal.innerHTML= 'Edit Company';
-        form_modal.action = '/companies/'+id;
-        name_edit.value = name;
-        address_edit.value = address;
-        type_method.innerHTML =method;
-        button.classList.remove('d-none');
-    }
+  function edit(id,name,address){
+    name_modal.innerHTML= 'Edit Company';
+    form_modal.action = '/companies/'+id;
+    name_edit.value = name;
+    address_edit.value = address;
+    type_method.innerHTML =method;
+    button.classList.remove('d-none');
+  }
 
-    function show(name,address)
-    {
-        name_modal.innerHTML= 'Show Company';
-        form_modal.action = ''
-        name_edit.value = name;
-        address_edit.value = address;
-        type_method.innerHTML ='';
-        button.classList.add('d-none');
-    }
-</script>
```

Comment:
They will have to wait for your branch [Feature#48/dashboard information #63](https://github.com/BC-MAY-21-ROR/kata-06-reloj-checador-k06-t01/pull/63) to merge, So that you can make a rebase to the updated master.

Since the changes of that branch will cause conflicts in this branch.